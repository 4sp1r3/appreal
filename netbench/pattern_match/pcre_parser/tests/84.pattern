/[\x61-\x7a]/i