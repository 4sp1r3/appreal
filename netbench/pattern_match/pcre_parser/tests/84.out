lex	: modif_front: SLASH
yacc	: pcre_delim <<< SLASH
yacc	: modif_front <<< pcre_delim
lex	: LBOX
yacc	: classStart <<< LBOX
lex	: CHARCLASS2VALUE HEX-HEX
lex	: a-z
yacc	: rangechars <<< INT 31329
yacc	: inclass_unit <<< rangechars
lex	: RBOX
yacc	: inclass_ext_unit <<< inclass_unit
yacc	: inclass <<< inclass_ext_unit
yacc	: classEnd <<< RBOX
yacc	: class <<< classStart inclass classEnd
yacc	: unit <<< class
lex	: SLASH
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: inslash <<< rv
yacc	: pattern <<< inslash
yacc	: pcre_delim <<< SLASH
yacc	: modif_rear_unit <<< MODIF_CASELESS
yacc	: modif_rear_ext <<< modif_rear_unit
yacc	: modif_rear <<< pcre_delim-modif_rear_ext
yacc	: pcre <<< modif_front pattern modif_rear
